integration-tests/src/test/java/fr/pilato/elasticsearch/crawler/fs/test/integration/FsCrawlerRestIT.java

```diff
@@ -74,6 +74,42 @@ public void testAllDocumentsWithRest() throws Exception {
         }
     }
 
+    @Test
+    public void testDocumentWithCustomTagsAndWithRest() throws Exception {
+        Path from = rootTmpDir.resolve("resources").resolve("documents");
+        if (Files.notExists(from)) {
+            staticLogger.error("directory [{}] should exist before wa start tests", from);
```

Comment:
`directory [{}] should exist before wa start tests` should be `directory [{}] should exist before we start tests`.